README.md

```diff
@@ -60,7 +60,7 @@ Then go ahead and reference the Paystack inline script in your index.html:
             <p>
               <PaystackButton
                 text="Make Payment"
-                class="payButton"
+                className="enterCssButtonStyle"
```

Comment:
This's a keyword name in react. `className` is used to specify a CSS class. https://reactjs.org/docs/dom-elements.html#classname